feat: adiciona acionamento automático dos LEDs RGB em sequência

JohnatasSilva opened the pull request:
Implementada a função `acionarLEDsEmSequencia

JohnatasSilva committed:
feat: adiciona acionamento automático dos LEDs RGB em sequência

JohnatasSilva committed:
feat: adiciona nova funcionalidade de acionamento automático dos LEDs

KayleKylian reviewed (approved):
Okay, à primeira vista, funcional.

JohnatasSilva committed:
Merge branch 'main' into feat/timer-auto-off

KayleKylian committed:
Update atividade.c

Só mudei o nome pra ficar padronizado, e tirei um protótipo extra que não tinha uso
